types/parse/index.d.ts

```diff
@@ -293,10 +294,19 @@ namespace Parse {
          * @returns Promise that is resolved with base64 data
          */
         getData(): Promise<string>;
+        url(options?: { forceSecure?: boolean }): string;
+        metadata(): any;
+        tags(): any;
```

Comment:
ref: https://parseplatform.org/Parse-SDK-JS/api/master/Parse.File.html

```suggestion
        tags(): Record<string, any>;
```